includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -405,9 +407,9 @@ public function process_payment( $order_id, $force_save_payment_method = false )
 		$order = wc_get_order( $order_id );
 
 		try {
-			$manual_capture = 'yes' === $this->get_option( 'manual_capture' );
+			$manual_capture = 'yes' === $this->get_option( 'manual_capture' ) ? Payment_Capture_Type::MANUAL() : Payment_Capture_Type::AUTOMATIC();
```

Comment:
this would make a nice function, `$this->get_capture_type()`

Reply:
Added in d267d83ae72a9784fb9c3462b0cbf78af439e84c . Nice catch! ❤️